dpnp/backend/kernels/dpnp_krnl_linalg.cpp

```diff
@@ -879,11 +886,29 @@ void func_map_init_linalg_func(func_map_t &fmap)
         eft_DBL, (void *)dpnp_inv_default_c<double, double>};
 
     fmap[DPNPFuncName::DPNP_FN_INV_EXT][eft_INT][eft_INT] = {
-        eft_DBL, (void *)dpnp_inv_ext_c<int32_t, double>};
+        get_res_type_with_aspect<>(),
+        (void *)dpnp_inv_ext_c<
+            int32_t, func_type_map_t::find_type<get_res_type_with_aspect<>()>>,
+        get_res_type_with_aspect<std::false_type>(),
+        (void *)dpnp_inv_ext_c<
+            int32_t, func_type_map_t::find_type<
+                         get_res_type_with_aspect<std::false_type>()>>};
     fmap[DPNPFuncName::DPNP_FN_INV_EXT][eft_LNG][eft_LNG] = {
-        eft_DBL, (void *)dpnp_inv_ext_c<int64_t, double>};
+        get_res_type_with_aspect<>(),
+        (void *)dpnp_inv_ext_c<
+            int64_t, func_type_map_t::find_type<get_res_type_with_aspect<>()>>,
+        get_res_type_with_aspect<std::false_type>(),
+        (void *)dpnp_inv_ext_c<
+            int64_t, func_type_map_t::find_type<
+                         get_res_type_with_aspect<std::false_type>()>>};
     fmap[DPNPFuncName::DPNP_FN_INV_EXT][eft_FLT][eft_FLT] = {
-        eft_DBL, (void *)dpnp_inv_ext_c<float, double>};
+        get_res_type_with_aspect<>(),
```

Comment:
Could you please double check that float needs to be converted to double, since I've got different result:
```
numpy.linalg.inv(numpy.arange(4, dtype=numpy.float32).reshape(2, 2)).dtype
dtype('float32')
```